includes/class-sensei-utils.php

```diff
@@ -1567,6 +1567,21 @@ public static function user_complete_course( $course_id = 0, $user_id = 0 ) {
 	}
 
 	/**
+	 * Checks if HTML security is enabled or disabled
+	 * @return boolean
+	 */
+	public static function html_security()
```

Comment:
Rename this to something more intention-revealing. Like `is_video_embed_html_sanitization_enabled`